ego/scenario_setting.json

```diff
@@ -6,8 +6,7 @@
     "recover": false,
     "result_id": null,
     "gridversion": "v0.4.2",
-    "csv_import": "data/PF_NEP_Qpnom"
-
+    "csv_import": "/data/ehv_grids/etrago_k10_t1"
```

Comment:
I had changed this i to not have a leading '/'. Don't know which way you prefer...